### ISSUE TITLE
Successfully on creating image, but nothing happen when plugged in RPI 2

### ISSUE THREAD

**Issue by msasongko** [spartacus06/rpi2-fedora-image-builder#13]
I had successfully on creating image Fedora-Minimal-armhfp-22-3, under Fedora 21 x64 bit running on VMWare Player.
Then, create image on Micro SD using Win32DiskImager.
After plugging in on RPI 2, when turn on, nothing boot happen.
The light red and green in RPI 2 lights continuously.
But, when using Raspbian, on booting, green light will be blinking.
Any clue to get this image works ?

**Comment by sjenning** [comment]
Hmm... I just retested and it seems to work fine for me.  Are you sure you are copying the .img file to the SD card and not the .raw file? The .raw file is just the decompressed version of the .xz that was downloaded from the repo and does not have the partition layout the RPi2 expects.

The format of the image should be something like this:

```
# parted Fedora-Minimal-armhfp-22-3-sda.raw.img print
Model:  (file)
Disk /root/rpi2-fedora-image-builder/Fedora-Minimal-armhfp-22-3-sda.raw.img: 1993MB
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start   End     Size    Type     File system  Flags
 1      1049kB  106MB   105MB   primary  fat16        lba
 2      106MB   1993MB  1887MB  primary  ext4
```

If that looks right, you might try mounting the partitions on another machine to make sure the contents of the boot partition (the first FAT partition) look correct.  Should have these files:
https://github.com/raspberrypi/firmware/tree/master/boot

**Comment by msasongko** [comment]
![oncreatingimage](https://cloud.githubusercontent.com/assets/13218955/8561271/22c51bc6-254d-11e5-8254-67ae062670ab.jpg)
![postcreatingimage](https://cloud.githubusercontent.com/assets/13218955/8561274/29e0d9ea-254d-11e5-98e1-590b80426739.jpg)
Thanks for your respond.
Here is attached, what I was doing, when creating image and validating after creating image.
I change settings.conf with BOOTSIZE = 200 and ROOTSIZE=3000.
Is there any way to validate my image ?
What should I do with https://github.com/raspberrypi/firmware/tree/master/boot ?

**Comment by sjenning** [comment]
It looks like you either don't have the latest version or have deleted some files.  The "scripts" directory is not present.  It contains the code for populating the boot partition.  Please:

```
git pull
git reset --hard origin/master
git clean -fd
./clean.sh
```

Then try again.

**Comment by bartmichu** [comment]
I think I had the same issue at first when I downloaded just the `mkimage.sh` file instead of the whole thing.

**Comment by msasongko** [comment]
Yes, I have made mistakes, eq : not creating scripts directory and copying all its contents.
After doing your recommendation, Fedora Minimal can be ran smoothly.
For your information, I have tried the same procedures, for Fedora Server and Fedora Workstation, but I could not get the same result.
For this issue, I think I will closed this issue.
Thanks for your thread, because it can make me starting to learn Linux with this tiny Raspbery Pi 2 B, from scratch.
